doc/whats_new.rst

```diff
@@ -25,6 +25,9 @@ New features
    - Added the :class:`ensemble.IsolationForest` class for anomaly detection based on
      random forests. By `Nicolas Goix`_.
 
+   - Added new supervised learning algorithm: :ref: `Multi-layer Perceptron <multi-layer-perceptron>`
```

Comment:
MLP module is still in dev so linking is not working right now, should I add the dev module ref or just remove the ref altogether?